tck/running_the_tck.asciidoc

```diff
@@ -22,6 +22,8 @@ The TCK uses `testng`
 
 == Dependencies
 
+The application server must make a MockTracer (https://github.com/opentracing/opentracing-java/tree/master/opentracing-mock) available to the application.
+
 To enable the tests in your project you need to add the following dependency to your build:
 
 [source, xml]
```

Comment:
Do you really want this type of detail in the spec document?  Looks like something that could easily get out of sync with the actual pom.xml files...

Reply:
Agreed, I'll do more research and potentially ask the mailing list, although for now it should be okay as it matches mpConfig.